liteflow-labs/starter-kit
# Redirection to metamask mobile

### Project organization
- Closes #51 
- Related to [trello issue](https://trello.com/c/xJUiC2YE/527-high-priority-fix-error-when-login-with-metamask-on-mobile)

### Description
Checks if `window.ethereum` is exists or not and depending on the return, if it's also mobile, it will now redirect user to metamask mobile app now.

### How to test
Visit the deployment link and try logging in on mobile device, it should redirect to playstore (tested on Android)

### Checklist
- [ ] Check on iphone
